pages/api/add-user.ts

```diff
@@ -0,0 +1,28 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { ResponseType } from '../../interfaces';
+import { prisma } from '../../lib/prisma';
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<ResponseType>
+) {
+  const { email } = req.body;
+
+  if (req.method === 'POST') {
+    await prisma.user
+      .create({
+        data: {
+          email: email,
+        },
+      })
+      .then(() => {
+        res
+          .status(200)
+          .json({ message: 'User is created', success: true, data: null });
```

Comment:
you shouldn't return null

Reply:
then what should we written in data??

Reply:
you should return the data you saved. @leewan-09 do you have better suggestion then please share

Reply:
It's can't be return as it is a void function I tried to do that way .

Reply:
why void function?

Reply:
may be I am awaiting it. I don't know exactly but I get an error while doing that.